### PR TITLE
Improve error reporting from vhd-util scan

### PR DESCRIPTION
**Pull request by MarkSymsCtx**

### CODE REVIEW

File: vhd/lib/vhd-util-scan.c [hunk 1]
```diff
@@ -1255,13 +1255,17 @@ vhd_util_scan_find_volume_targets(int cnt, char **names,
 	targets   = NULL;
 
 	err = lvm_scan_vg(volume, &vg);
-	if (err)
+	if (err) {
+		fprintf(stderr, "lvm_scan_vg failed %s\n", err);
```
Comment (edwintorok) [inline vhd/lib/vhd-util-scan.c:1259]: shouldn't this be %d?

**Review by edwintorok — CHANGES_REQUESTED** [review]

File: vhd/lib/vhd-util-scan.c (hunk 1 above)
Comment (MarkSymsCtx) [reply vhd/lib/vhd-util-scan.c]: Yes and it failed to compile, I obviously updated this after the last compile. Fixed now.

**Review by TimSmithCtx — APPROVED** [review]

**Review by edwintorok — APPROVED** [review]

File: vhd/lib/vhd-util-scan.c (hunk 1 above)
Comment (edwintorok) [reply vhd/lib/vhd-util-scan.c]: might be useful to add some github actions to try and build blktap, but getting all the dependencies would be tricky, they probably won't be neatly available in distro packages.

Comment (MarkSymsCtx) [reply vhd/lib/vhd-util-scan.c]: The dependencies won't be available at all in the distros, as they require the frankentap kernel device and headers which only exist in the Citirx Hypervisor kernel patchqueue.

**Pull request merged**